docs/source/developing.rst

```diff
@@ -23,3 +23,5 @@ To get started, work through the following sections of the documents. You will n
 * Build the various Runestone servers and run them
 
 If you are interested in deploying a production version of Runestone to serve books, begin by setting up a development environment. Once you are confident in that process, the :ref:`moving to production instructions<moving-to-production>` will help you set up a production ready server.
+
+To get started with getting the Runestone code go to the following page :ref:`<https://runestone-monorepo.readthedocs.io/en/latest/structure.html>`. 
```

Comment:
We should avoid hardcoding the actual URL to the docs and use the format of `:ref:` as is used on line 25 above.

Reply:
@solijonovam This means you should change this to a relative link reference rather than an absolute link reference which would break if the pages were restructured.  Basically, you are being asked to match what was done on line 25, which is a relative reference.